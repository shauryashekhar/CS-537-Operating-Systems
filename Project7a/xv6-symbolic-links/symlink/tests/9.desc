Creation of symlink using absolute path (in a different directory)
