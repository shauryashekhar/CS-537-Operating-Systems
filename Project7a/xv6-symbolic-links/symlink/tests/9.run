cd xv6; ../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_9 | grep SYMLINK; cd ..
