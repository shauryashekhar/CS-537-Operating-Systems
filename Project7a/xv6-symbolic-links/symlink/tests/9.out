SYMLINK: Symlink return = 0
SYMLINK:.              1 21
SYMLINK:..             1 1
SYMLINK:testSYM9.txt   2 20
